app/components/app_icon.stories.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React from 'react';
+
+import {storiesOf} from '@storybook/react-native';
+import {withKnobs, select, number} from '@storybook/addon-knobs';
+
+import AppIcon from './app_icon';
```

Comment:
As I said before, this component no longer exists.

Reply:
I've used a different component as the demo component.